### PR TITLE
feat(stellar): script for requesting testnet funds

### PR DESCRIPTION
**Pull request by jcs47**
https://axelarnetwork.atlassian.net/browse/AXE-6456

### CODE REVIEW

File: stellar/faucet.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+if (require.main === module) {
+    const program = new Command();
+
+    program
+        .name('faucet')
+        .addOption(new Option('--recipient <recipient>', 'recipient to request funds for'))
+        .addOption(
+            new Option(
+                '--minBalance <amount>',
```
Comment (ahramy) [inline stellar/faucet.js:41]: We want to move towards a consistent naming scheme. The CLI flags/naming should be kebab case than camel case. Please refer `gateway.js` or `deploy-contract-js` in Stellar Folder.

```suggestion
                '--min-balance <amount>',
```

File: stellar/faucet.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const { Command, Option } = require('commander');
```
Comment (ahramy) [inline stellar/faucet.js:1]: Could you also update `README.md` file regarding to how to use stellar faucet?

File: stellar/faucet.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    await horizonServer.friendbot(recipient).call();
+
+    printInfo('Funds requested', recipient);
+}
+
+async function mainProcessor(options, processor) {
+    const config = loadConfig(options.env);
+    const chain = getChainConfig(config, options.chainName);
+    await processor(chain, options);
+    saveConfig(config, options.env);
```
Comment (ahramy) [inline stellar/faucet.js:32]: Does this script saves balances and sequence to config? if not, we should remove `saveConfig`.

Comment (jcs47) [reply stellar/faucet.js]: good point, removed this

**Pull request merged**